survey-runner-database/cloudwatch.tf

```diff
@@ -6,10 +6,10 @@ resource "aws_cloudwatch_metric_alarm" "database_storage_alert" {
   namespace           = "AWS/RDS"
   period              = "300"
   statistic           = "Minimum"
-  threshold           = "1073741824"
+  threshold           = "${var.database_free_storage_alert_level * 1024 * 1024 * 1024}"
```

Comment:
It would be but AWS don't expose the storage as a percentage and the terraform scripts don't know the mappings between instance type and available memory.